### PR TITLE
BVTCK-189 Disable the JavaFX tests by default and provide an option to enable them

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/BVTCK-189

### CODE REVIEW

**Comment by gsmet** [comment]
Force pushed an update to the documentation to be more in line with what @gunnarmorling said in the JIRA issue.

**Comment by gsmet** [comment]
@aguibert interested in your feedback on this one. I merged the other changes to master so you can just get this branch.

File: documentation/src/main/asciidoc/configuration.asciidoc [hunk 1]
```diff
@@ -159,4 +159,19 @@ Another interesting property is
...
+[[configuration-javafx]]
+
+The Bean Validation specification mandates a support of JavaFX if JavaFX is available in the classpath.
+
+While JavaFX is included in the Oracle JDK and some other JDKs include https://wiki.openjdk.java.net/display/OpenJFX/Main[OpenJFX],
+it might not be included in all JDKs.
+
+Having JavaFX available in a container environment might also not be straightforward.
+
+For these reasons, the JavaFX tests are disabled by default when running the TCK with the default options.
```
Comment (gunnarmorling) [inline documentation/src/main/asciidoc/configuration.asciidoc]: Redundant "default": ... are disabled when running the TCK with the default options.

**Comment by aguibert** [comment]
hi @gsmet, I checked out this branch locally and everything is working as expected.  LGTM

File: documentation/src/main/asciidoc/configuration.asciidoc (hunk 1 above)
Comment (gsmet) [reply documentation/src/main/asciidoc/configuration.asciidoc]: Fixed.

**Comment by gsmet** [comment]
@aguibert thanks for confirming. I merged it and will release a new TCK shortly.

**Comment by gsmet** [comment]
@aguibert I released 2.0.2.Final today.

Thanks for reporting these issues and for testing the PRs, greatly appreciated.

**Pull request closed without merging**